### ISSUE TITLE
Syntax fields not saved first time when having multiple layouts

### ISSUE THREAD

**Issue by spinn** [rainlab/pages-plugin#494]
- OctoberCMS Build: 473
- RainLab Pages Plugin Version: 1.4.7
- PHP Version: 8.0.7

### Description:
As the title says, when you hit "save" on a new page for the first time, syntax fields might not get saved when you have multiple layouts. This only happens when the layout is not the first in the "layout" dropdown (setting default 0/1 has no direct effect on this if the default layout is the first in the dropdown). When you hit "save" for the second time (and all times after that, even when editing), it always gets saved correctly.

### Steps To Reproduce:
Create multiple layouts, each with a different syntax field, eg.:

#### layout1.htm
```
description = "Layout1"

[staticPage]
==
{variable name="var1" type="text" label="Var 1"}{/variable}

Var: {{ var1 }}

{% page %}
```

#### layout2.htm
```
description = "Layout2"

[staticPage]
==
{variable name="var2" type="text" label="Var 2"}{/variable}

Var: {{ var2 }}

{% page %}
```

When creating a new page, choose the layout that is **not** pre-selected, enter something in the syntax field and hit save once. After previewing the saved page, the syntax field will not get displayed, and when inspecting the new page (content/static-pages/filename.htm), the variable will not be present in the viewBag. After pressing save for the second time, it will appear.

**Comment by daftspunk** [comment]
Hi @spinn 

This should be fixed by 1014c90fd6e09427c011e746b8c94387bab9acbf

Please test and report back, then we can ship it as a release.

Thanks!

**Comment by spinn** [comment]
Hi @daftspunk,

tested and now everything works as expected.

Thanks for the quick fix!
